src/commands/run-tests.yml

```diff
@@ -14,6 +14,11 @@ parameters:
       folders using a colon-separated or semicolon-separated list.
     type: string
     default: ''
+  select-by-name:
+    description: >
+      Name of the test used to select test suite elements, specified as a path relative to the project root folder. To create a test suite, the command uses only the test elements that match the specified name. Use the wildcard character (*) to match any number of characters. Use the question mark character (?) to match a single character.
```

Comment:
Test names don't get specified as file paths. We also don't create a suite with this information, but filter one down.

We should also tell folks how they specify multiple names (by separating them by space).

Something like:

"Name of test used to select test suite elements, specified as a list of names separated by
      spaces."

Reply:
Yes I have updated the description accordingly.